examples/migraphx_docker/README.md

```diff
@@ -0,0 +1,9 @@
+# Building the MIGraphX Dockerfile
+
+You can build the provided dockerfile with the following command:
+
+    docker build -f Dockerfile.hip-clang -t migraphx .
```

Comment:
May be we can remove the dockerfile here, you can just refer to the docker files in the top level directory.

Reply:
Switched